### DISCUSSION TITLE
Help with settings

### DISCUSSION THREAD

**Discussion by cnmining** [cagnulein/qdomyos-zwift#1286] [Q&A] [answered]
Sorry I must be dumb and can't figure this out. I updated to the mtb setting but feel very little difference if any. When I adjust the +\- resistance I also don't really feel anything.

I'm on an elite direto XR and riding a 1x10 mtb with 30t front... also a big guy lol.

What area should I be adjusting settings if I'm hoping to make it feel harder in each of the gears so I have a bit more range? Most seem to download and feel their issue solved. So I'm clearly missing something easy.

**Comment by cagnulein** [comment, marked as answer]
@cnmining you have to use the gears tile not the resistance one. Let me know

**Reply by cnmining** [reply]
Ohh that wasn't on the default page for me but I see how to add it. I will go play with that. Thank you for the fast reply.

**Comment by cagnulein** [comment]
no problem! i'm here if you need me (actually i will be here since i'm going to sleep right now :) )

**Reply by cnmining** [reply]
Worked awesome, you rock!
